src/Abstractions/NexusMods.Abstractions.Loadouts.Synchronizers/ALoadoutSynchronizer.cs

```diff
@@ -895,13 +897,76 @@ private bool ShouldWinWrapper(IDb db, in GamePath path, in SyncNodePart existing
         // Return the newer one
         return ShouldWin(path, existingLoadoutItem, replacementLoadoutItem);
     }
-    
+
+    /// <summary>
+    /// Returns true if <paramref name="a"/> should win over <paramref name="b"/> based
+    /// on rules defined in a collection.
+    /// </summary>
+    protected Optional<bool> ShouldOtherWinWithCollectionRules(LoadoutItem.ReadOnly a, LoadoutItem.ReadOnly b)
```

Comment:
Comment and function name don't seem match. `ShouldWin` returns true if the second parameter should win, while this function does the opposite while using same naming